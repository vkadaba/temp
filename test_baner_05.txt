{
     a = + b c;
     c = + d b;
     e = < a 1;
 }
