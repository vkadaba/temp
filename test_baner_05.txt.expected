a: int #
b: int #
e: bool #
c: int #
d: int #
